.github/workflows/ci.yml

```diff
@@ -8,15 +8,40 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stack-yaml:
+          - stack.yaml           # ghc-9.0
+          - stack-lts-18.28.yaml # ghc-8.10
+          - stack-lts-16.31.yaml # ghc-8.8
+          - stack-lts-14.27.yaml # ghc-8.6
+          - stack-lts-12.26.yaml # ghc-8.4
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: freckle/stack-cache-action@v2
+        with:
+          stack-yaml: ${{ matrix.stack-yaml }}
+      - uses: freckle/stack-action@v3
+        with:
+          stack-yaml: ${{ matrix.stack-yaml }}
+
+  nightly:
+    runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
-      - uses: freckle/stack-cache-action@v1.0.1
-      - uses: freckle/stack-action@v2
+      - uses: freckle/stack-cache-action@v2
+        with:
+          stack-yaml: stack-nightly.yaml
+      - uses: freckle/stack-action@v3
         with:
-          weeder: false
-          hlint: false
+          stack-yaml: stack-nightly.yaml
+          stack-arguments: --resolver nightly
 
-  hlint:
+  lint:
```

Comment:
I was gonna ask if `stack-action` does this but it looks like it doesn't anymore. We may want to update the readme since it still says it lints.

Reply:
Hmm, I thought I made that clear in the latest README,

![](https://files.pbrisbin.com/screenshots/screenshot.3124565.png)